Example/ObjC/Bulletin/TextFieldBulletinPage.h

```diff
@@ -13,16 +13,16 @@
 @interface TextFieldBulletinPage : BLTNPageItem
 
 /// The block executed when the user finished inputting the text.
-@property (nonatomic, nullable) void (^textInputHandler)(TextFieldBulletinPage *, NSString * _Nullable);
+@property (nonatomic, nullable) void (^textInputHandler)(TextFieldBulletinPage * _Nonnull, NSString * _Nullable);
 
 /// The text field presented by the page.
-@property (nonatomic, strong, readonly) UITextField *textField;
+@property (nonatomic, strong, readonly) UITextField * _Nonnull textField;
```

Comment:
This property is nullable, since it's only assigned in `-[TextFieldBulletinPage makeViewsUnderDescriptionWithInterfaceBuilder:]`.

```suggestion
@property (nonatomic, strong, readonly, nullable) UITextField *textField;
```